admin/src/useModelsState.js

```diff
@@ -53,7 +54,12 @@ const buildEnState = async () => {
 
   if (Array.isArray(singleTypes) && isValidLength(singleTypes)) {
     const data = await getItems(singleTypes, request);
-    const enLocaleData = getEnLocaleData(data);
+    const enLocaleData = getEnLocaleData(data).map((enSingleTypeData)=> {
+      return {...enSingleTypeData, 
+              fullResults: [enSingleTypeData?.fullResults?.[0]], 
```

Comment:
Maybe use slice to make i clear we take array and return array
```suggestion
              fullResults: enSingleTypeData?.fullResults?.slice(0,1), 
```